test_scripts/Polices/appID_Management/ATF_P_TC_OnAppPermissionChanged_appRevoked_true.lua

```diff
@@ -0,0 +1,70 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+--     [ChangeRegistration]: appRevoked:true
+-- Description:
+-- <appID> application is assigned with specific policies (for example, appID = 123abc->
+-- "app_policies" -> "123abc" : { <specific policies> }
+-- Steps:
```

Comment:
Under description we should have the following sections: 

1. Used preconditions
--here list all Preconditons such as the one listed in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/98